src/relayserver/TxStoreManager.js

```diff
@@ -78,7 +78,7 @@ class TxStoreManager {
     return this.txstore.asyncRemove({
       $and: [
         { 'nonceSigner.nonce': nonce },
-        { 'nonceSigner.signer': signer }]
+        { 'nonceSigner.signer': signer.toLowerCase() }]
```

Comment:
Actually then you need toLowerCase() all insertions as well I think.

Reply:
better `web3.utils.toChecksumAddress()` on the signer itself, instead of lowering it all the time.

Reply:
Will lowercase. this is not a good place to fail on checksum